features/fout-meldingen.feature

```diff
@@ -86,6 +86,30 @@ Rule: type van parameter waarde is correct
         | name       | reason                                     |
         | huisnummer | waarde 'A' is geen getal tussen 1 en 99999 |
 
+Rule: parameter waarde voldoet aan de opgegeven validaties van de parameter
+
+    Scenario: parameter waarde is kleiner dan de gedefinieerde minimum waarde
+    Als '/wozobjecten?postcode=8000GB&huisnummer=1&page=0' wordt aangeroepen
+    Dan bevat de response de volgende kenmerken
+        | naam   | waarde                                              |
+        | title  | Een of meerdere parameters zijn niet correct.       |
+        | status | 400                                                 |
+        | detail | Parameter 'page' bevat een (deels) ongeldige waarde |
+    En bevat de response de volgende invalidParams
+        | name | reason                                                     |
+        | page | waarde '0' is kleiner dan de toegestane minimum waarde (1) |
```

Comment:
vanuit automatiseerbaarheid van de feature snap ik de aanhalingstekens. Maar is dat correct bij een integer?

Reply:
De aanhalingstekens zijn niet toegevoegd voor automatisering. Het betreft tekst die getoond wordt in de UI. Als het leesbaarder is, kunnen de aanhalingstekens wat mij betreft weg. Dit is vanuit UX geredeneerd. In dit geval is het voor de hand liggend, maar hoe zou je het doen bij een string? Bij een string kan je zonder aanhalingstekens niet meer zien wat hoort bij de string. Vooral als daar ook nog spaties voor of achter staan.

Reply:
bij een string moeten er in elk geval aanhalingstekens omheen. wat mij betreft is dit niet echt een probleem